tests/testthat/test-writing_sheet_data.R

```diff
@@ -168,7 +168,7 @@ test_that("Writing sheetData rows XML - iris", {
     "<row r=\"151\"><c r=\"A151\" t=\"n\"><v>5.9</v></c><c r=\"B151\" t=\"n\"><v>3</v></c><c r=\"C151\" t=\"n\"><v>5.1</v></c><c r=\"D151\" t=\"n\"><v>1.8</v></c><c r=\"E151\" t=\"s\"><v>7</v></c></row>"
   )
 
-  for (i in 1:length(expected_rows)) {
+  for (i in seq_along(expected_rows)) {
```

Comment:
I personally would have replaced it with `seq_len(length(...))`. yours does look better, but I've been bitten enough not to blindly trust `seq_along` for anything not data.frame related.